packages/opencensus-web-instrumentation-zone/test/test-interaction-tracker.ts

```diff
@@ -494,8 +494,19 @@ describe('InteractionTracker', () => {
       const xhr = new XMLHttpRequest();
       xhr.onreadystatechange = noop;
       spyOn(xhr, 'send').and.callFake(() => {
+        // Simulate the actual monkey-patch done on `Send()`.
```

Comment:
Is there a way to avoid having to duplicate the monkey-patch logic here?

For example, can you do something like `const realSend = xhr.send` and then `xhr.send = () => { realSend(); ....}`?

Reply:
Isn't that mean that the actual HTTP requests will be sent? Isn't it something we want to avoid to call the actual `send`?

Reply:
Another option would be to export a function that runs the logic on the monkey-patch and called from the test.

Reply:
Chatted offline - I like the idea of an exported function.